low-level/framework/messaging/egress_accumulated_msgs_processor.py

```diff
@@ -120,20 +122,32 @@ def run(self):
                     if isinstance(message, bytes):
                         message = message.decode()
                     dict_msg = json.loads(message)
-                    if "actuator_response_type" in dict_msg["message"]:
-                        event_time = dict_msg["message"] \
-                            ["actuator_response_type"]["info"]["event_time"]
-                        time_diff = int(time.time()) - int(event_time)
-                        if time_diff > self.MSG_TIMEOUT:
-                            continue
-                    if "sensor_response_type" in dict_msg["message"]:
-                        logger.info(f"Publishing Accumulated Alert: {message}")
-                    if isinstance(self._producer, MessageProducer):
-                        self._producer.send([message])
-                        logger.info(f"Published Accumulated Message {message}")
-                        self.store_queue.delete()
+                    if dict_msg.get("iem"):
+                        try:
+                            EventMessage.send(
```

Comment:
Ok so IEM needs different handling now, given their enablers and respective APIs are different